sql/sql_repl.cc

```diff
@@ -2809,6 +2809,8 @@ void mysql_binlog_send(THD* thd, char* log_ident, my_off_t pos,
 
   /* Check if the dump thread is created by a slave with semisync enabled. */
   thd->semi_sync_slave = is_semi_sync_slave();
+  if (thd->semi_sync_slave)
+    sql_print_information("Semi-sync replication enabled on slave.");
```

Comment:
I don't think we need this. The existing informational messages already allude to whether or not semi-sync is enabled on slave. That is, the message starting with `Start semi-sync binlog_dump to slave...` will only be displayed when the slave has semi-sync enabled.